<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="QCheckBox" name="cbEnable">
     <property name="text">
      <string>有効化</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>アセットリスト</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTreeView" name="tvAssetsList"/>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btAddUnitypackage">
       <property name="text">
        <string>unitypackageを追加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btRemoveUnitypackage">
       <property name="text">
        <string>選択のunitypackageを削除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
